static/bidder-info/theadx.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://ssp.theadx.com/request?pbs=1"
+maintainer:
+  email: "ssp@theadx.com"

```

Comment:
@mustafakemal16 We have sent a verification email to ssp@theadx.com. Please reply to that email with 'Received' to complete the verification process. Thank you!